lib/rules/no-get.js

```diff
@@ -124,19 +147,20 @@ module.exports = {
           const path = node.arguments[0].value;
           context.report({
             node,
-            message: makeErrorMessageForGet(path, false),
+            message: makeErrorMessageForGet(path, false, useOptionalChaining),
             fix(fixer) {
-              if (path.includes('.')) {
+              if (path.includes('.') && !useOptionalChaining) {
                 // Not safe to autofix nested properties because some properties in the path might be null.
                 return null;
               }
 
-              if (!isValidJSVariableName(path)) {
+              if (!isValidJSPath(path)) {
                 // Do not autofix since the path would not be a valid JS variable name.
```

Comment:
This comment should also replace `variable name` with `path`.

Reply:
Fixed.